packages/foundation-models/package.json

```diff
@@ -20,24 +20,27 @@
     "internal.d.ts"
   ],
   "scripts": {
-    "compile": "tsc",
+    "compile": "pnpm run generate-zod && tsc",
     "compile:cjs": "tsc -p tsconfig.cjs.json",
     "test": "NODE_OPTIONS=--experimental-vm-modules jest",
     "lint": "eslint \"**/*.ts\" && prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -c",
     "lint:fix": "eslint \"**/*.ts\" --fix && prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -w --log-level error",
-    "check:public-api": "node --loader ts-node/esm ../../scripts/check-public-api-cli.ts"
+    "check:public-api": "node --loader ts-node/esm ../../scripts/check-public-api-cli.ts",
+    "generate-zod": "ts-to-zod src/openai/openai-types.ts src/openai/openai-types-schema.ts && sed -i 's|export const|\\/**\\n * @internal\\n *\\/\\nexport const|' src/openai/openai-types-schema.ts"
```

Comment:
[pp] This is a bit too much bash magic for my taste, is there a more explicit way without me reading the sed man?

Reply:
(feel free to ask others, I am a real command line noob)

Reply:
😆 This is due to the `check:public-api` script. The generated schema does not contain `@internal` but still needs to be exported. The library `ts-to-zod` also does not support adding this kind of things. I looked into the check public api script, but it seems also quite cumbersome to support excluding compiled schema file.

Reply:
yes it does. we will improve this in the future